### ISSUE TITLE
Why the learning rate keeps growing？

### ISSUE THREAD

**Issue by mmmeiko7** [Tianxiaomo/pytorch-YOLOv4#466]
Why the learning rate in the log starts at 1.6e-10, then grows to the lr we set(0.0261) and keeps the same?
